src/test/java/com/google/sps/api/CalendarClientHelperTest.java

```diff
@@ -14,85 +14,76 @@
 
 package com.google.sps.api;
 
-import com.google.sps.servlets.SampleServlet;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.EventAttendee;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
-import org.mockito.Mockito;
 import java.io.IOException;
-import java.io.PrintWriter;
-import java.io.StringWriter;
-import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
-import com.google.api.services.calendar.model.Event;
-import com.google.api.services.calendar.model.EventAttendee;
-import javax.management.Attribute;
 
 @RunWith(JUnit4.class)
 public final class CalendarClientHelperTest {
 
   private static final String ACCEPTED = "accepted";
   private static final String DECLINED = "declined";
-  private final CalendarClientHelper calendarClientHelper = new CalendarClientHelper();
-  private static EventAttendee ATTENDING_SELF = new EventAttendee();
-  private static EventAttendee DECLINED_SELF = new EventAttendee();
-  private static EventAttendee NOT_RESPONDED_SELF = new EventAttendee();
-  private static EventAttendee ATTENDING_GUEST = new EventAttendee();
-  private static EventAttendee DECLINED_GUEST = new EventAttendee();
-  private static EventAttendee NOT_RESPONDED_GUEST = new EventAttendee();
+  private EventAttendee ATTENDING_SELF = new EventAttendee();
+  private EventAttendee DECLINED_SELF = new EventAttendee();
+  private EventAttendee NOT_RESPONDED_SELF = new EventAttendee();
+  private EventAttendee ATTENDING_GUEST = new EventAttendee();
+  private EventAttendee DECLINED_GUEST = new EventAttendee();
+  private EventAttendee NOT_RESPONDED_GUEST = new EventAttendee();
 
-  public CalendarClientHelperTest() {
-    ATTENDING_SELF.setSelf(true);
-    DECLINED_SELF.setSelf(true);
-    NOT_RESPONDED_SELF.setSelf(true);
-    ATTENDING_SELF.setResponseStatus(ACCEPTED);
-    ATTENDING_GUEST.setResponseStatus(ACCEPTED);
-    DECLINED_SELF.setResponseStatus(DECLINED);
-    DECLINED_GUEST.setResponseStatus(DECLINED);
-  }
+
+
+  @Before
+  public void setAttendees(){
+      ATTENDING_SELF.setSelf(true);
```

Comment:
In this method you should also run code like `ATTENDING_SELF = new EventAttendee();`. Otherwise, these objects might get modified on different properties than the ones you set in this method.

Reply:
and then as a result you should renamte it to be just camel case since it's not a constant anymore.